From f7d74d7cc6e197f42ed3270a51c01ce54d0d5003 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 4 Nov 2019 15:16:30 +0700
Subject: [PATCH 105/328] arm64: dts: renesas: hihope-common: enable vspm_if,
 vspi, vspb

This commit enables vspb, vspi, vspm_if support

Signed-off-by: Son Lam <son.lam.ym@renesas.com>
Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 8dcb15b..357f01b 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -111,6 +111,10 @@
 		label = "rcar-sound";
 		dais = <&rsnd_port0>;
 	};
+
+	vspm_if {
+		compatible = "renesas,vspm_if";
+	};
 };
 
 &audio_clk_a {
@@ -459,3 +463,11 @@
 		};
 	};
 };
+
+&vspb {
+	status = "okay";
+};
+
+&vspi0 {
+	status = "okay";
+};
-- 
1.9.1

